extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -51,10 +57,29 @@ def timestamp() -> str:
     return timestamp
 
 
+def send_to_trigger(data: typing.Dict[str, str]) -> int:
+    return requests.post(_trigger_url(), data=data).status_code
+
+
+def trigger(data: typing.Optional[typing.Dict[str, str]] = None) -> int:
+    if data is None:
```

Comment:
With this salt+signature generation if nothing is sent to trigger seems to me would result in anyone being able to trigger without needing to know the secret key, am I just missing something?

Reply:
Or is `flask.request.form` not `None` even if nothing is sent in the request?

Reply:
I'm pretty sure that in that case `flask.request.form` will be an empty `dict`, but I've improved the `BAD_REQUEST` related tests a bit.